src/back/TicketManager/src/PaymentService/TicketManager.PaymentService.Contracts/Payments/PaymentTokenResponse.cs

```diff
@@ -3,4 +3,4 @@ namespace TicketManager.PaymentService.Contracts.Payments;
 public class PaymentTokenResponse
 {
     public Guid Token { get; set; }
```

Comment:
Specification requires this field to be named `id`

Reply:
Addressed in 8a8011da.